syntax = "proto3";
package protos.playbooks;

import "protos/base.proto";
import "google/protobuf/wrappers.proto";
import "google/protobuf/struct.proto";


message SqlDataFetchTask {
  google.protobuf.StringValue database = 1;
  google.protobuf.StringValue query = 2;
  google.protobuf.StringValue order_by_column = 3;
  google.protobuf.UInt64Value limit = 4;
  google.protobuf.UInt64Value offset = 5;
}